compiler/src/dmd/attrib.d

```diff
@@ -701,8 +701,8 @@ extern (C++) class ConditionalDeclaration : AttribDeclaration
 extern (C++) final class StaticIfDeclaration : ConditionalDeclaration
 {
     ScopeDsymbol scopesym;          /// enclosing symbol (e.g. module) where symbols will be inserted
-    private bool addisdone = false; /// true if members have been added to scope
-    private bool onStack = false;   /// true if a call to `include` is currently active
+    public bool addisdone = false; /// true if members have been added to scope
+    public bool onStack = false;   /// true if a call to `include` is currently active
```

Comment:
symbols are `public` by default, so it is sufficient to simply remove `private`, though this obviously works too.